langchain/callbacks/tracers/stdout.py

```diff
@@ -154,9 +158,18 @@ def _on_tool_end(self, run: Run) -> None:
 
     def _on_tool_error(self, run: Run) -> None:
         crumbs = self.get_breadcrumbs(run)
-        print(
+        self.function_callback(
             f"{get_colored_text('[tool/error]', color='red')} "
             + get_bolded_text(f"[{crumbs}] [{elapsed(run)}] ")
             + f"Tool run errored with error:\n"
             f"{run.error}"
         )
+
+
+class ConsoleCallbackHandler(FunctionCallbackHandler):
```

Comment:
is main diff between this and StdOutCallbackHandler that it'll print on every type of event?

Reply:
ah see that ConsoleCallbackHandler predates this pr, so not relevant here!